src/main/java/nl/hsac/fitnesse/fixture/slim/HttpTest.java

```diff
@@ -151,6 +163,38 @@ protected boolean postToImpl(String body, String serviceUrl) {
         result = postProcessResponse();
         return result;
     }
+
+    protected boolean postFileToImpl(String fileName, String serviceUrl) {
+        boolean result;
+        resetResponse();
+        String url = getUrl(serviceUrl);
+        String fitnesseFilesDir;
+        String filePath;
+
+        try {
+            fitnesseFilesDir  = getEnvironment().getFitNesseFilesSectionDir();
+            filePath = String.format("%s/%s", fitnesseFilesDir, fileName);
+            File file = new File(filePath);
+            if(!isFile(file)) {
```

Comment:
I believe we should be consistent with the way paths to upload files are handled by BrowserTest:
if the file is located in the files directory its path must beging with 'files/', otherwise we expect a full path.

That means we always use this method, and not start by looking in the files section. I.e. remove the 4 lines above.